packages/libs/client/src/signing/tests/signWithChainweaver.test.ts

```diff
@@ -122,4 +141,37 @@ describe('signWithChainweaver', () => {
       { sig: 'transfer-key-sig' },
     ]);
   });
+
+  it('Wallet signs but does not have the signer key and returns sig null', async () => {
```

Comment:
The `it` already refers to the `describe('signWithChainweaver')`. Also, we're not testing the wallet but the `signWithChainweaver` function.

```suggestion
  it('signs but does not have the signer key and returns sig null', async () => {
```